modules/react/src/components/export-video/export-video-panel-preview.tsx

```diff
@@ -215,7 +216,7 @@ export class ExportVideoPanelPreview extends Component<
             >
               <DeckGLOverlay
                 ref={this.deckRef}
-                glOptions={{stencil: true}}
+                deviceProps={{type: 'webgl', webgl: {stencil: true}}}
```

Comment:
@chrisgervang `stencil` was no longer recognized here.

Reply:
`Object literal may only specify known properties, and 'stencil' does not exist in type 'WebGLContextProps'.ts(2353)
device.d.ts(145, 5): The expected type comes from property 'webgl' which is declared here on type 'CreateDeviceProps'`